### PR TITLE
sci-electronics/librepcb: bump version to 0.1.7

### PR DESCRIPTION
**Pull request by ktrace**
- bump version
- disable broken test

Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Victor Kustov <ktrace@yandex.ru>

### CODE REVIEW

**Comment by AndrewAmmerlaan** [comment]
@ktrace Since this package is currently lacking a maintainer, would you be interested in proxy-maintaining it in ::gentoo? Or maybe maintaining it in ::guru?

https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers
https://wiki.gentoo.org/wiki/Project:GURU

**Comment by ktrace** [comment]
> @ktrace Since this package is currently lacking a maintainer, would you be interested in proxy-maintaining it in ::gentoo? Or maybe maintaining it in ::guru?

Yeah, that would be interesting.

**Comment by AndrewAmmerlaan** [comment]
> > @ktrace Since this package is currently lacking a maintainer, would you be interested in proxy-maintaining it in ::gentoo? Or maybe maintaining it in ::guru?
> 
> Yeah, that would be interesting.

Great! 

To proxy-maintain it here in ::gentoo you can add yourself to the `metadata.xml` file like this:
```
  <maintainer type="person" proxied="yes">
    <email>YOUR EMAIL</email>
    <name>YOUR NAME</name>
  </maintainer>
  <maintainer type="project" proxied="proxy">
    <email>proxy-maint@gentoo.org</email>
    <description>Gentoo Proxy Maintainers</description>
  </maintainer>
```

Or if you prefer to maintain it directly you can open an access request to the ::guru repository as described [here](https://wiki.gentoo.org/wiki/Project:GURU/Information_for_Contributors). In which case we will remove the package from ::gentoo so you can add it to ::guru instead.

I emerged your ebuild and got two small test failures, otherwise this looks great. Build log is attached here: 
[librepcb-0.1.7:20221018-120550.log](https://github.com/gentoo/gentoo/files/9810395/librepcb-0.1.7.20221018-120550.log)

File: sci-electronics/librepcb/metadata.xml [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+   <maintainer type="project" proxied="proxy">
+     <email>sci-electronics@gentoo.org</email>
+     <description>Gentoo Electronics Project</description>
+   </maintainer>
+   <maintainer type="person" proxied="yes">
+     <email>ktrace@yandex.ru</email>
+     <name>Victor V. Kustov</name>
+   </maintainer>
+   <longdescription lang="en">
+     LibrePCB is a free EDA software to develop printed circuit boards.
```
Comment (AndrewAmmerlaan) [inline sci-electronics/librepcb/metadata.xml:13]: Looks good :+1: 

We've just got one complaint from the QA bot::
```
RedundantLongDescription | metadata.xml longdescription closely matches DESCRIPTION
```
We should either just drop the long description, or replace it with a longer one.

**Review by AndrewAmmerlaan — CHANGES_REQUESTED** [review]

File: sci-electronics/librepcb/metadata.xml (hunk 1 above)
Comment (ktrace) [reply sci-electronics/librepcb/metadata.xml:13]: ready

**Comment by AndrewAmmerlaan** [comment]
Nice work, Thanks :+1: 

I adjusted the warning a bit so it mentions the upstream bug report.

**Pull request closed without merging**